### ISSUE TITLE
Tests failing for paperless-ng

### ISSUE THREAD

**Issue by j0hax** [NixOS/nixpkgs#136626]
### Describe the bug
Trying to set up paperless-ng for the first time leads to tests, and ultimately the build, failing.

### Steps To Reproduce
1. set `services.paperless-ng.enable = true`
2. run `nixos-rebuild switch`

### Expected behavior
Paperless-ng is built and started

### Log File
[paperless-ng.log](https://github.com/NixOS/nixpkgs/files/7108334/paperless-ng.log)

### Notify maintainers

@earvstedt

@Flakebi

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
 - system: `"x86_64-linux"`
 - host os: `Linux 5.10.61, NixOS, 21.11 (Porcupine)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3.15`
 - channels(johannes): `""`
 - channels(root): `"nixos-21.11pre312722.9df8a8489e3, nixos-hardware"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos
```

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute: paperless-ng
# a list of nixos modules affected by the problem
module: services.paperless-ng
```

**Comment by j0hax** [comment]
Same Issue appears in [Hydra](https://hydra.nixos.org/build/151893234)

**Comment by Flakebi** [comment]
Weird, the build that failed on hydra works fine for me (`/nix/store/ccyx3f12nmym1v4n17f5s5gffc8ldqpz-paperless-ng-1.5.0`).

cc @erikarvstedt

**Comment by erikarvstedt** [comment]
Same here, `paperless` (`x86_64-linux`) succeeds locally with the current nixpkgs master (d431839ab4494499714f2b6f001413fe380607eb).

**Comment by j0hax** [comment]
Paperless-ng works (in praxis) fine for me if I set `doCheck = false;`

Is it possible that the test fails due to differing hardware specs (limited RAM or `/tmp` size)?

**Comment by erikarvstedt** [comment]
Fixed in #158118.
